### PR TITLE
Bump version to v1.18.0

### PR DESCRIPTION
**Pull request by bvtujo**
### Change Summary
* Bump version in preparation for transition to CD.
* Update release notes. 
* Remove producing gitsha binaries.

----
Enter `[N/A]` in the box, if an item is not applicable to your change.

**Testing**
- [x] Unit tests passed

**Documentation**
- [ x] Contacted our doc writer
- [n/a ] Updated our README
----

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.17.0
\ No newline at end of file
+1.18.0
```
Comment (hencrice) [inline VERSION]: can we get rid of the newline? We cat this file in buildspec and I did not test with an extra newline

File: ecs-cli/modules/version/version.go [hunk 2]
```diff
@@ -22,7 +22,7 @@ package version
 // repository. Only the 'Version' const should change in checked-in source code
```
Comment (hencrice) [inline ecs-cli/modules/version/version.go]: Is this comment still accurate?

File: VERSION (hunk 1 above)
Comment (bvtujo) [reply VERSION]: Addressed in most recent commit. Will squash before merging. Took a couple tries but what finally got it was `perl -pi -e 'chomp if eof' VERSION`

File: ecs-cli/modules/version/version.go (hunk 2 above)
Comment (bvtujo) [reply ecs-cli/modules/version/version.go]: I don't believe so. We've been manually bumping the version (I think) for a while. I modeled this PR off the last one @SoManyHs merged in.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 1.18.0
+* Enhancement - Print verbose messages for credential chain errors (# 937)
```
Comment (sonofachamp) [inline CHANGELOG.md]: Does the space in `# 937` result in a broken link?

**Pull request merged**